src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/IPAOpenDataServiceImpl.scala

```diff
@@ -128,8 +128,12 @@ object IPAOpenDataServiceImpl {
   private def getDescription(institutionKind: InstitutionKind, mapped: Map[String, Int], record: List[RecordValue]) =
     institutionKind match {
       case InstitutionKind.Agency => record(mapped(InstitutionsFields.agencyDescription)).select[String]
-      case InstitutionKind.AOO    => record(mapped(InstitutionsFields.aooDescription)).select[String]
-      case InstitutionKind.UO     => record(mapped(InstitutionsFields.uoDescription)).select[String]
+      case InstitutionKind.AOO    =>
+        Some(s"${record(mapped(InstitutionsFields.aooDescription))
+            .select[String]} - ${record(mapped(InstitutionsFields.agencyDescription)).select[String]}")
```

Comment:
`.select[]` returns and `Option`. This string concatenation would result in the string
`Some(AOO name) - Some(Institution Name)`